src/main/java/com/jalasoft/selenium/giovanny/movies/RentalStatement.java

```diff
@@ -0,0 +1,142 @@
+package com.jalasoft.selenium.giovanny.movies;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Class that lists and displays the
+ * rentals amount and the frequent renter points.
+ *
+ * Created by xianqe on 4/25/2017.
+ */
+public class RentalStatement {
+    private String name;
+    private List<Rental> rentals = new ArrayList<Rental>();
+    private double totalAmount;
+    private int frequentRenterPoints;
```

Comment:
instead of saving as attributes of class
you can have two methods to calculateTotalAmount and calculateTotalFrequentRenterPoints.
if you do that, it will not be necessary to have the clearTotals method. :)